next-tavla/src/Admin/scenarios/Boards/components/TagModal/AddExistingTag.tsx

```diff
@@ -0,0 +1,43 @@
+import { TTag } from 'types/meta'
+import { useBoardsSettings } from '../../utils/context'
+import { difference, uniq } from 'lodash'
+import { ActionChip } from '@entur/chip'
+import { AddIcon } from '@entur/icons'
+import { Heading3 } from '@entur/typography'
+
+function AddExistingTag({
+    addTag,
+    boardTags,
+}: {
+    addTag: (tag: TTag) => void
+    boardTags: TTag[]
+}) {
+    const { boards } = useBoardsSettings()
+
+    const allTags = uniq(
+        boards.flatMap((board) => {
+            return board?.meta?.tags ?? []
+        }),
+    )
+
+    const suggestedTags = difference(allTags, boardTags)
+
+    return (
+        <>
+            <Heading3>Legg til eksisterende merkelapp</Heading3>
+            <div className="flexWrap g-1">
+                {suggestedTags.map((tag) => (
+                    <ActionChip
+                        key={tag}
+                        onClick={() => addTag(tag)}
+                        aria-label="Legg til merkelapp"
+                    >
+                        {tag} <AddIcon />
+                    </ActionChip>
+                ))}
+            </div>
+        </>
```

Comment:
Er det noe spesiell grunn til at dette må være et fragment?

Reply:
Siden de begge er barn av en flex med col-direction. Alternativet hadde vært å lage det om til en div og sette flex-properties på nytt. Hadde jo sikkert sett bedre ut i DOM-treet, men også lagt til enda mer "boilerplate"-kode for samme resultat. Hva tenker du?